inc/application/view/extension/datahelpersextension.class.php

```diff
@@ -55,6 +55,7 @@ public function getFilters(): array {
          new TwigFilter('safe_html', [$this, 'getSafeHtml'], ['is_safe' => ['html']]),
          new TwigFilter('verbatim_value', [$this, 'getVerbatimValue']),
          new TwigFilter('shortcut', [$this, 'underlineShortcutLetter'], ['is_safe' => ['html']]),
+         new TwigFilter('enhanced_html', [RichText::class, 'getEnhancedHtml']),
```

Comment:
With this, using `|raw` filter will not be necessary anymore.
```suggestion
         new TwigFilter('enhanced_html', [RichText::class, 'getEnhancedHtml'], ['is_safe' => ['html']),
```